omod/pom.xml

```diff
@@ -196,4 +216,56 @@
 		</plugins>
 	</build>
 
+	<!-- different profiles for building/testing against different version of 
```

Comment:
Do we need to duplicate the profiles in this omod/pom.xml in addition to the root pom.xml?